common/robotPositioner.h

```diff
@@ -0,0 +1,186 @@
+
+#pragma once
+
+// c++ standard includes
+#include <vector>
+#include <list>
+#include <cstdio>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <chrono>
+#include <csignal>
+#include <thread>
+#include <algorithm>
+
+#include "../third_party/units.h"
+#include "../robots/tank.h"
+
+using namespace std::literals;
+using namespace units;
+using namespace units::length;
+using namespace units::time;
+using namespace units::velocity;
+using namespace units::literals;
+using namespace units::angle;
+using namespace units::constants;
+using namespace units::dimensionless;
+using namespace units::angular_velocity;
```

Comment:
In general you should never ``using namespace`` in the global namespace as there's no way for other bits of code to 'undo' it - in this case the solution is to move ``RobotPositioner`` into the  ``BoBRobotics::Robots`` namespace (and move it into the robots folder)

Reply:
Also I don't think you need this one:
```
using namespace units;
```
I think the other ``using``s cover this